packages/codec/lib/format/index.ts

```diff
@@ -80,7 +80,18 @@
  *     the raw data that led to the error.  See [[Format.Errors|`Format.Errors`]] for
  *     more information.
  *
- *   5. `reference`: This field is a debugger-only feature and does not
+ *   5. `interpretations`: This field will also be present when `kind` is equal
+ *     to `"value"`; it is an object that may contain additional information
+ *     about the decoded value beyond what is found in `value`.  (All fields in
+ *     `interpretations` are always optional and will be included only when
+ *     applicable.)  You may wonder, what is the difference between `value`
+ *     and `interpretations`?  The answer is that the distinction is largely
+ *     historical; `interpretations` was not originally part of the format, and
```

Comment:
I'm not sure this is the best place to include this note about historical reasoning... is there somewhere else we can note it? Since it's not immediately relevant here, I'd wager.

Reply:
Where else would it go?  This sort of overview seems like the most sensible place to me...

Reply:
It just seems like a detail that should come later. But I don't feel strongly enough to keep this comment unresolved.